### PR TITLE
add advertisements pagination

### PR DESCRIPTION
**Pull request by AndreyShchukin**

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```
Comment (vnikitash) [inline .idea/inspectionProfiles/Project_Default.xml:1]: .idea should be in gitignore. Please exclude this file from commit

File: .idea/olx2.iml [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <component name="NewModuleRootManager">
```
Comment (vnikitash) [inline .idea/olx2.iml:3]: .idea should be in gitignore. Please exclude this file from commit

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vnikitash) [inline .idea/vcs.xml:1]: .idea should be in gitignore. Please exclude this file from commit

File: app/Http/Controllers/AdvertisementController.php [hunk 4]
```diff
@@ -23,13 +24,10 @@ public function __construct(AdvertisementService $advertisementService)
...
-    public function index(GetAdvertisementRequest $request)
+    public function index()
     {
-        $category_id = Arr::get($request->validated(), 'category_id');
-
-        $list = $this->advertisementService->getListOfAdvertisements($category_id);
-
-        return response()->json(['data' => $list]);
+        $advertisements = Advertisement::paginate(5);
+        return view('advertisements', compact('advertisements'));
```
Comment (vnikitash) [inline app/Http/Controllers/AdvertisementController.php:30]: before return empty line please

File: app/Http/Controllers/AdvertisementController.php [hunk 5]
```diff
@@ -23,13 +24,10 @@ public function __construct(AdvertisementService $advertisementService)
         return $this->advertisementService = $advertisementService;
     }
 
-    public function index(GetAdvertisementRequest $request)
+    public function index()
```
Comment (vnikitash) [inline app/Http/Controllers/AdvertisementController.php:27]: add return type like public function index(): some_type

File: database/seeds/AdvertisementsSeeder.php [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+class AdvertisementsSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        for ($i=0; $i < 100; $i++) {
```
Comment (vnikitash) [inline database/seeds/AdvertisementsSeeder.php:14]: you are making insert in loops. that means 100 mysql requests. Please build 1 array of 100 items and make ONE insert with using of this array

File: database/seeds/CategoriesSeeder.php [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<?php
```
Comment (vnikitash) [inline database/seeds/CategoriesSeeder.php:1]: Empty seeder. do we need it?

File: supervisord.log [hunk 8]
```diff
@@ -13,3 +13,60 @@
 2020-06-12 16:23:51,305 INFO success: php-fpmd entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
```
Comment (vnikitash) [inline supervisord.log:13]: add this file to gitignore and remove it from the repo

File: database/seeds/AdvertisementsSeeder.php (hunk 6 above)
Comment (AndreyShchukin) [reply database/seeds/AdvertisementsSeeder.php:14]: Redid as a factory

File: database/seeds/CategoriesSeeder.php (hunk 7 above)
Comment (AndreyShchukin) [reply database/seeds/CategoriesSeeder.php:1]: For the future)

File: app/Http/Controllers/AdvertisementController.php (hunk 4 above)
Comment (AndreyShchukin) [reply app/Http/Controllers/AdvertisementController.php:30]: Done

File: .idea/vcs.xml (hunk 3 above)
Comment (AndreyShchukin) [reply .idea/vcs.xml:1]: Done

File: .idea/olx2.iml (hunk 2 above)
Comment (AndreyShchukin) [reply .idea/olx2.iml:3]: Done

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (AndreyShchukin) [reply .idea/inspectionProfiles/Project_Default.xml:1]: Done

File: supervisord.log (hunk 8 above)
Comment (AndreyShchukin) [reply supervisord.log:13]: Done

File: app/Http/Controllers/AdvertisementController.php (hunk 5 above)
Comment (AndreyShchukin) [reply app/Http/Controllers/AdvertisementController.php]: Done

**Pull request merged**